src/components/my-journey/Stepper.tsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
`Line 51` remove unnecessary redundancy  and make it like this `areAllSubStepsTrue(step) || (index === 0 || areAllSubStepsTrue(steps[index - 1]))
`